packages/aws-cdk/lib/api/stack-events/stack-activity-monitor.ts

```diff
@@ -282,520 +325,3 @@ export class StackActivityMonitor {
     return path;
   }
 }
-
-function padRight(n: number, x: string): string {
-  return x + ' '.repeat(Math.max(0, n - x.length));
-}
-
-/**
- * Infamous padLeft()
- */
-function padLeft(n: number, x: string): string {
-  return ' '.repeat(Math.max(0, n - x.length)) + x;
-}
-
-function calcMaxResourceTypeLength(template: any) {
-  const resources = (template && template.Resources) || {};
-  let maxWidth = 0;
-  for (const id of Object.keys(resources)) {
-    const type = resources[id].Type || '';
-    if (type.length > maxWidth) {
-      maxWidth = type.length;
-    }
-  }
-  return maxWidth;
-}
-
-interface PrinterProps {
```

Comment:
Rest of this file moved to various files in `cli/activity-printer` but also got refactored a lot